### ISSUE TITLE
Does this code set the learning rate？when i use my own dataset，the loss becomes nan and i want to decrease the learning-rate but i cant find it

### ISSUE THREAD

**Issue by zhangzhang-vvv** [jaungiers/LSTM-Neural-Network-for-Time-Series-Prediction#77]

**Comment by nicolauga** [comment]
Same question here!
183/321 [================>.............] - ETA: 24s - loss: 0.0052
184/321 [================>.............] - ETA: 23s - loss: nan   
185/321 [================>.............] - ETA: 23s - loss: nan
186/321 [================>.............] - ETA: 23s - loss: nan
187/321 [================>.............] - ETA: 23s - loss: nan

**Comment by VincieD** [comment]
No, it takes the defaulf one by adam. If you want to change it, add it in the config.json and change it in the model.py

**Comment by dongdongrj** [comment]
Change the LR by below code:
K.set_value(model.optimizer.lr, 0.0001)

**Comment by xFireBerg** [comment]
Also faced similar issues, but then realized the preprocessing work is insufficient, and null values in any part of your dataset can break the model, thus giving you nan losses.
